### ISSUE TITLE
[FR] Gantt board support

### ISSUE THREAD

**Issue by moolight-seashell** [AppFlowy-IO/AppFlowy#3392]
### Description

i have not see this anywhere in issues or roadmap so ther is =)

### Impact

allow to manage project

**Comment by annieappflowy** [comment]
I'm glad you submitted this request :D

**Comment by joshuamsalazar** [comment]
Agree!

**Comment by officialsuyogdixit** [comment]
I need this!

**Comment by Kribbage** [comment]
Would be great to have! would love to have something that supports all the basic PM features including gantt that doesn't require me to sign up for a SaaS

**Comment by InJJWeTrust** [comment]
Nice to see that this was added to Planned for 2024. 

This is one of those high-value features for anyone who is looking for a Project Management tool with the ability to also have documentation in the same place. It would definitely positively affect Appflowy's attractiveness in my opinion at least.

Even just implementing it as different type of calendar view would make sense to me, as it is essentially a calendar view in it's roots, just showing tasks beginning and end dates, and having task dependencies. 

Really looking forward to this being added as once it is, I will immediately shift over to Appflowy for my small-business as well as recommend it to others in similar agile and traditional software development teams.

**Comment by CovetingEpiphany2152** [comment]
I believe in Notion this is called [timeline view](https://www.notion.com/help/timelines). 
I'd like to add on that I think the dependency arrows are very useful for project planning. Some tools lack the ability to show said dependencies, and some only support one-to-one dependency. Ideally, it would be great if appflowy can support many-to-many dependencies, since tasks can always have multiple dependencies.

**Comment by casainho** [comment]
We are in 2025 and I need this for simple project management!!

**Comment by Jabe64** [comment]
Yes we need this too with my team to switch from Monday to AppFlowy.

@annieappflowy is this still planned?

And shouldn't the [open tickets tagged 2024](https://github.com/AppFlowy-IO/AppFlowy/issues?q=state%3Aopen%20label%3A%222024%22%20) be set to 2025 or even 2026? Will the roadmap be updated? 

It's not very clear what's planned. I can see what's for [Q1 2025](https://github.com/AppFlowy-IO/AppFlowy/issues?q=state%3Aopen%20label%3A%22Q1%2025%22%20), what's for [Q2 2025](https://github.com/AppFlowy-IO/AppFlowy/issues?q=state%3Aopen%20label%3A%22Q2%2025%22%20) but I don't understand if what was planned for 2024 is planned as a priority or for Q3 2025 or even after that?

Please let me know.

**Comment by dudewti** [comment]
Would very much like an uppdate on this @annieappflowy. Having calendar, kanban and gantt in one place is hard to find and is the single thing keeping me from switching over my entire teams project management.

**Comment by Hula1267** [comment]
Whats going on with this?

**Comment by Ray0m** [comment]
I'm here to add some juice to this request as well!
